geogebra-scalable.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" version="1">
```

Comment:
But it's also incorrect. Compare the Flathub logo
![](https://raw.githubusercontent.com/flathub/org.geogebra.GeoGebra/master/geogebra-256.png)
with the one on Wikipedia
![](https://upload.wikimedia.org/wikipedia/commons/thumb/5/57/Geogebra.svg/256px-Geogebra.svg.png)

It's an SVG and does not really contain a resolution. This is the file I uploaded rendered at 256px
<img src="https://raw.githubusercontent.com/FakeShemp/org.geogebra.GeoGebra/master/geogebra-scalable.svg?sanitize=true" width="256"/>
It can be scaled to render at any resolution you want.